frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -107,27 +108,45 @@ const InstallNeedsApprovalMessage: React.FC<InstallNeedsApprovalMessageProps> =
   approve,
 }) => {
   const { t } = useTranslation();
+
+  const canPatchInstallPlans = useAccessReview({
+    group: InstallPlanModel.apiGroup,
+    resource: InstallPlanModel.plural,
+    namespace,
+    verb: 'patch',
+  });
```

Comment:
This is also declared in `frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx`

Reply:
Is it better practice to declare it once and export it? I haven't worked with these very often.

Reply:
yes, we should make our code DRY (Dont Repeat Yourself)

Reply:
I gave it a go, but it looks like since it's a hook, it will violate the rules of hooks if I turn it into a shared function.